go.sum

```diff
@@ -607,6 +612,7 @@ github.com/prometheus/procfs v0.6.0/go.mod h1:cz+aTbrPOrUb4q7XlbU9ygM+/jj0fzG6c1
 github.com/prometheus/tsdb v0.7.1/go.mod h1:qhTCs0VvXwvX/y3TZrWD7rabWM+ijKTux40TwIPHuXU=
 github.com/quobyte/api v0.1.8/go.mod h1:jL7lIHrmqQ7yh05OJ+eEEdHr0u/kmT1Ff9iHd+4H6VI=
 github.com/remyoudompheng/bigfft v0.0.0-20170806203942-52369c62f446/go.mod h1:uYEyJGbgTkfkS4+E/PavXkNJcbFIpEtjt2B0KDQ5+9M=
+github.com/robertkrimen/otto v0.0.0-20200922221731-ef014fd054ac/go.mod h1:xvqspoSXJTIpemEonrMDFq6XzwHYYgToXWj5eRX1OtY=
```

Comment:
Did you see who pulled this in?

I guess it's a transitive dependency somewhere, but it wasn't obvious from the vendor changes below. I guess adding a random javascript parsing library is just standard modern programming process :-/

Reply:
From what I recall, otto is used in K8s to script CSI behavior in e2e tests, maybe that's where it came from

Reply:
Fair enough, thanks.

/lgtm
/approve